app/views/libraries/index.html.erb

```diff
@@ -1,7 +1,11 @@
 <h1>Libraries</h1>
+<%= link_to "Sort by total number of books", "/libraries?sort=book_count" %>
 <% @libraries.each do |library| %>
 <h4><%= library.name %> </h4>
   <ul>
+    <% if params[:sort] == "book_count" %>
```

Comment:
This is by no means wrong but I'm not sure why you have this line? If you have time could you shed more light on this?